### PR TITLE
I have added contributing.md file and code_of_conduct.md file along with some very minor changes in the Readme.rst file

### PR DESCRIPTION
**Pull request by VanshajPoonia**
I have added contributing.md file and code_of_conduct.md file along with some very minor changes in the Readme.rst file

In the Contributing.md, I have left some space to add the support email and some steps on how to run the project locally in order to contribute.
I request the maintainers to complete the details and merge them with the main project. It will ease the process of contribution to the project.

### CODE REVIEW

**Comment by Gallaecio** [comment]
I read “amazing” in the README changes and stopped reading…

What is the source of those documents, where did you copy them from?

**Comment by VanshajPoonia** [comment]
> I read “amazing” in the README changes and stopped reading…
> 
> What is the source of those documents, where did you copy them from?

I got them from a source that was expected to be discovered by the "members" a long ago. And getting offended by the word "amazing" isn't a sign of a mentally stable person brother. It was to make the readers familiar with, and not unsmiling.
Don't mind but I tried to make the repo more readable. CODE_OF_CONDUCT is a standard file that you can find in any good project and contributing.md required "some" effort to write and took a little more effort to make to match the standard format, but it seems that I am giving the right explanation but to a wrong person.
I have already got my Hacktoberfest swags so it clearly means that I was contributing to this repo for a sake of interest but never expected to get such a response from an Open source community.
Feels like I wasted my time and energy.

**Comment by VanshajPoonia** [comment]
> > Feels like I wasted my time and energy.
> 
> I am sorry you feel that way.
> 
> However, I don’t think we can merge your changes without further discussion and changes. Introducing a new code of conduct into a project is not something trivial.
> 
> You could have copied [the Scrapy one](https://github.com/scrapy/scrapy/blob/master/CODE_OF_CONDUCT.md), and since the maintainers and much of the community is shared that would have probably been accepted. But instead, you submitted a different (similar?) code of conduct, without attribution, and you included into the same pull request unrelated changes.
> 
> If you are willing to put more time, please considering splitting unrelated changes (readme, contributing, code of conduct) into separate pull requests, so that they can be discussed separately.
> 
> If you have lost interesting in getting these changes merged, please feel free to close this pull request.

I also apologize for my reply.
Since everything is cool now, I think I should make separate PRs and I hope we can discuss them and make the required changes :D
Also, are you being sarcastic while saying that I should have copied Scrapy's CODE_OF_CONDUCT or you are kinda serious? I am confused xD as I just realized the code of conduct is the same. I thought its a quite common template, in fact, I got it from a blog for my own project that I planned to make open source LOL. I think you should celebrate that your CODE_OF_CONDUCT file is quite famous.

So now, could you please guide me that what should my immediate next step?

**Comment by Gallaecio** [comment]
I think the immediate next step would be to remove the Contributor document and revert the README changes (you can create a separate pull request for each of those), and replace the contents of the code of conduct with those of Scrapy’s, which I think we can use *as is*, without any further change.

If you look at the end of Scrapy’s, you will see that it was not actually been written by us. That is what I mean originally when I asked about the source of your code of conduct, it is usually polite, when not also a legal requirement, to include proper attribution in copied content.

**Comment by Gallaecio** [comment]
> I think we can use _as is_

Alternatively, maybe we can upgrade to the [latest version](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) of the upstream code of conduct.

**Comment by VanshajPoonia** [comment]
> > I think we can use _as is_
> 
> Alternatively, maybe we can upgrade to the [latest version](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) of the upstream code of conduct. Any of the 2 options works for me.

I am always inclined towards upgrades. Should I open a pull request with the same file of the CODE_OF_CONDUCT available at contributor covenant?

**Comment by VanshajPoonia** [comment]
I have adapted the CODE_OF_CONDUCT from the [here](https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md)

Let's discuss the changes required

**Review by Gallaecio — APPROVED** [review]
Looks good to me, thanks!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (kmike) [inline CODE_OF_CONDUCT.md:63]: I think we should insert contact method here :) We can use the same as in Scrapy's CoC:

```suggestion
[opensource@zyte.com](mailto:opensource@zyte.com)
```

**Comment by kmike** [comment]
Thanks @VanshajPoonia!

**Pull request merged**